src/Meshes/geometry.ts

```diff
@@ -915,6 +931,8 @@ export class Geometry implements IGetSetVerticesData {
             }
             this._vertexArrayObjects = {};
         }
+
+        // TODO: Do the meshes too?

```

Comment:
what do you mean?

Reply:
Was just thinking that we should invalidate mesh-vaos as well but it ended up in Mesh._rebuild instead.